hack/update-codegen.sh

```diff
@@ -70,7 +70,7 @@ pkg/api
 pkg/api/v1
 $(
   cd ${KUBE_ROOT}
-  find pkg/apis -name types.go | xargs dirname | sort
+  find pkg/apis -name types.go | xargs -I {} dirname {} | sort
```

Comment:
Good catch, thank you. It comes from https://github.com/kubernetes/kubernetes/pull/36164, removing.